src/bin/p-s.test.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path'
+import test from 'ava'
+import tester from 'cli-tester';
+import fs from 'fs';
+import os from 'os';
+import {createHash} from 'crypto';
+
+
+const babel = require.resolve('babel-cli/bin/babel-node');
+const cli = require.resolve('./p-s');
+
+
+const hash = string => createHash('md5').update(string).digest('hex');
+const tmpConfig = string => path.join(os.tmpdir(), `tmp-config-${hash(string)}.js`);
+const writeConfig = (file, string) => new Promise(resolve => fs.writeFile(file, string, resolve));
```

Comment:
actually writing real file is the only way to go. Mock-fs/memory-fs do not work, since testing is done in a separate process.